doc/source/tutorial/getting_started_frontera.rst

```diff
@@ -0,0 +1,48 @@
+.. _Frontera:
+
+Getting Started: Frontera
+=========================
+
+..  code-block:: bash
+
+  # Assuming the default Intel stack is loaded only boost and hdf5 modules are required
+  module load boost hdf5
+
+  # Build Grackle (optional)
+  # Following https://grackle.readthedocs.io/en/latest/Installation.html
+  mkdir -p ~/src
+  cd ~/src
+  git clone https://github.com/grackle-project/grackle
+  cd grackle
+  git submodule update --init
+  ./configure
+  # create build directory as install target later
+  mkdir build-icc
+  cd src/clib
+  # Adjust config to  set install path, and optimization options
+  # TACC Stampede config works just fine for TACC Frontera
+  sed -i 's#$(HOME)/local#$(PWD)/../../build-icc#;s/-xCORE-AVX2/-xCORE-AVX512/' Make.mach.tacc-stampede-intel
+  make machine-tacc-stampede-intel
+  make
+  make install
+
+  # Build Charm++
+  cd ~/src
+  git clone https://github.com/UIUC-PPL/charm.git
+  cd charm
+  git checkout v7.0.0
```

Comment:
Maybe we should just encourage people to download the latest version of charm++ from the main website http://charm.cs.uiuc.edu/software and mention that we last tested this with version 7.0.0? (So that we don't need to update this going forward?)

Reply:
I am currently leaning towards keeping it this way as this is a "tested" configuration known to work.
If there are future release of Charm++ (which may not work for whatever reason), downloading an older version from the main website may be more complicated than checking out a fixed tag. For example, the main website only offers the 7.0.0 tgz without mentioning previous releases).
Having said that, I don't have strong feeling about this so we can also update if you like.

Reply:
Sounds sensible to me.